### ISSUE TITLE
Unexpected token ReduxToastr.js

### ISSUE THREAD

**Issue by tomasfrancisco** [mxstbr/react-boilerplate#1370]
Getting this exact error related to react-redux-toastr: https://github.com/diegoddox/react-redux-toastr/issues/85

How can I make this running?

Thanks in advance!

**Comment by gihrig** [comment]
The problem seems to be a conflict of philosophy regarding speed of build vs distribution of ES Next code in production.

react-redux-toaster uses the ES6 spread operator [here](https://github.com/diegoddox/react-redux-toastr/blob/master/src/reducer.js#L37) in distribution code.

As suggested [here](https://github.com/diegoddox/react-redux-toastr/issues/85#issuecomment-266073613) this requires transpilation from ES Next to ES5. in order to work in most current environments.

Conversely, react-boilerplate does not attempt to transpile all of `node_models` as this will dramatically increase the build time for the project.

Bottom line is you can't have it both ways. Given that conflict, I think the [solution](https://github.com/diegoddox/react-redux-toastr/issues/85#issuecomment-266073613) given by  @dcmdestello is a reasonable one.

**Comment by redgenie** [comment]
Why don't you allow it to be transpiled in ES5? In you `webpack.base.js` include it as such:
```javascript
loaders: [{
      test: /\.js$/, // Transform all .js files required somewhere with Babel
      loader: 'babel',
      exclude: /node_modules(?!\/react-redux-toastr)/,
      query: options.babelQuery,
    },
```
This will exclude all `node_modules` except `react-redux-toastr`

**Comment by gihrig** [comment]
@redgenie That's an even better idea, thanks for contributing 🎉

**Comment by karloespiritu** [comment]
I encountered the same issue integrating `react-redux-toastr` with react-boilerplate and this is how I fixed it. In `internals/webpack.base.babel.js`, I updated to following lines near the end of the file, I switched the sequence of `jsnext:main` and `main`:

Before
```
    mainFields: [
      'browser',
      'jsnext:main',
      'main'
    ]
```

After (fixed version):

```
    mainFields: [
      'browser',
      'main',
      'jsnext:main'
    ]
```

**Comment by zilet** [comment]
There is a precompiled version available. 

Just use '/lib' in imports:

`import { actions as toastrActions } from 'react-redux-toastr/lib';`

**Comment by mayacode** [comment]
@zilet you saved my day!

**Comment by Mansourawy** [comment]
@zilet cool workaround ty
